content/en/platform/corda/5.2/developing-applications/cordapp-template/utxo-ledger-example-cordapp/running-the-chat-cordapp/_index.md

```diff
@@ -79,7 +78,7 @@ curl -X 'POST' \
   -H 'Content-Type: application/json' \
   -d '{
   "clientRequestId": "create-1",
-  "flowClassName": "com.r3.developers.csdetemplate.utxoexample.workflows.CreateNewChatFlow",
+  "flowClassName": "com.r3.developers.cordapptemplate.utxoexample.workflows.CreateNewChatFlow",
```

Comment:
On this page, it has `v5_1` in few places (lines 75 and 110, if I'm not wrong) - the links should still work I suppose, but we might want to have everything updated to `v5_2`